README.md

```diff
@@ -178,6 +178,14 @@ There are two optional parameters for logging, they are:
 * `log_file`: A file path to a logfile to log to.
 * `log_level`: Set the logging verbosity, from most to least: `debug`,`error`,`panic`
 
+### Overlapping Ranges
+
+The overlapping ranges feature is enabled by default, and will not allow an IP address to be re-assigned across two different ranges which overlap. However, this can be disabled.
+
+* `enable_overlapping_ranges`: *(boolean)* Checks to see if an IP has been allocated across another range before assigning it (defaults to `true`).
```

Comment:
would `enable_overlapping_ranges` be turned off if there is no VLAN ID?

Reply:
Currently, no logic here to enforce or otherwise detect that...

...probably if I wanted to turn it on/off for a given vlan ID, I'd probably create an additional configuration as a `net-attach-def` and refer to it along with any vlan ID specification you have in there.